pkg/pharos/cmd/setup.go

```diff
@@ -0,0 +1,49 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/fatih/color"
+	configpkg "github.com/lob/pharos/pkg/pharos/config"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+// Declare variables to be used as flags.
+var awsProfile string
+var awsRoleARN string
+var pharosURL string
+
+// setupCmd is the pharos setup command.
+var setupCmd = &cobra.Command{
+	Use:   "setup",
+	Short: "Setup Pharos config",
+	Long:  "Setup Pharos configuration file. Overwrites previously saved configuration.",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		return runSetup(pharosConfig, pharosURL, awsProfile, awsRoleARN)
+	},
+}
+
+func runSetup(pharosConfig string, url string, profile string, arn string) error {
+	c, err := configpkg.New(pharosConfig)
+	if err != nil {
+		return errors.Wrap(err, "unable to create reference to config file")
+	}
+
+	c.BaseURL = url
```

Comment:
What do you think of changing these values to only be set if they're not an empty string (`""`, the default value)? For example, this would allow us to update the `aws-profile` while leaving the URL the intact (by running `pharos setup --aws-profile production`)
```suggestion
	if c.BaseUrl != "" {
		c.BaseURL = url
	}
```

Reply:
i can dig it, i actually did this at first but wasn't sure if we wanted to overwrite the file